azure-pipelines.yml

```diff
@@ -152,34 +152,38 @@ jobs:
           Contents: '**/*'
         condition: always()
 
-  # - job: Danger
-  #   dependsOn: Build
-  #   steps:
-  #     # No need for checkout since we're using build artifacts.
-  #     - checkout: none
-
-  #     - template: azure-pipelines.artifacts.yml
-  #       parameters:
-  #         artifact: Build-PR-$(Build.BuildNumber)
-  #         buildId: $(Build.BuildId)
-
-  #     - template: azure-pipelines.tools.yml
-
-  #     - script: |
-  #         yarn perf:test
-  #       workingDirectory: packages/fluentui/perf-test
-  #       displayName: Perf Test
-
-  #     - script: |
-  #         if [ $BUILD_REASON == "PullRequest" ] && [ ! -z $DANGER_GITHUB_API_TOKEN ]; then
-  #           yarn danger ci
-  #         elif [ $BUILD_REASON != "PullRequest" ]; then
-  #           echo "Skipping Danger JS because BUILD_REASON ${BUILD_REASON} != PullRequest"
-  #         else
-  #           echo "Skipping Danger JS because API token wasn't found"
-  #         fi
-
-  #     - script: |
-  #         yarn danger ci
-  #       env:
-  #         DANGER_GITHUB_API_TOKEN: $(DANGER_GITHUB_API_TOKEN)
+  - job: Danger
```

Comment:
This is the 4th parallelized job after build - wondering what the queuing times would look after this (mostly seeing if we need to up the count of vms)

Reply:
I will be moving this to the `perf-test` pipeline as part of this PR, it won't affect gated CI time.

Reply:
(This is a Draft PR not ready for review.)